department-of-veterans-affairs/va.gov-cms
# VACMS-4212: Add past events bulk ops view.

## Description
See #4212 
Per convo with @swirtSJW & @kevwalsh , decision made is to create an events tab with an exposed filter to sort by date. This view (for now) is only visible to administrators. 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/107557805-10796100-6ba8-11eb-91b6-ed57920955e6.png)

## QA steps

- As an administrator, go to `admin/content/events/past` and visually verify (by clicking into individual events) that only past items are present.
- Confirm that filters and bulk op archive ops work as expected.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
